src/llm/servable.cpp

```diff
@@ -37,7 +39,7 @@
 
 namespace ovms {
 absl::Status GenAiServable::loadRequest(std::shared_ptr<GenAiServableExecutionContext>& executionContext, const ovms::HttpPayload& payload) {
-    SPDLOG_LOGGER_DEBUG(llm_calculator_logger, "Request body: {}", payload.body);
+    logRequestBody(payload.parsedJson);
```

Comment:
How about we expand that to `logRequestDetails(payload)` and inside that function we always log request body and URI.
Then here we could call such method if log level is debug or trace.